playback/tape_cassettes/s3/s3_tape_cassette.py

```diff
@@ -200,7 +201,14 @@ def iter_recording_ids(self, category, start_date=None, end_date=None, metadata=
 
             def content_filter(recording_str):
                 recording_metadata = decode(recording_str)
-                return all(recording_metadata.get(k) == v for k, v in metadata.items())
+                for k, v in metadata.items():
+                    if isinstance(v, str):
```

Comment:
This won't necessarily work for python 2 - you need to use `basestring`

![image](https://user-images.githubusercontent.com/9438326/104565550-1a657e00-5655-11eb-8aa2-4d56d8a4a180.png)